framework/test/unit/modules/token/commands/transfer_cross_chain.spec.ts

```diff
@@ -310,18 +338,6 @@ describe('CCTransfer command', () => {
 				},
 			);
 
-			await userStore.save(
```

Comment:
Do you want to explain the outcome of this removal? To make sure we are aligned here.